velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -167,19 +177,55 @@ class S3FileSystem::Impl {
     }
   }
 
-  // Configure default AWS credentials provider chain.
+  // Configure an AWSCredentialsProvider with access key and secret key.
+  std::shared_ptr<Aws::Auth::AWSCredentialsProvider>
+  getAccessKeySecretKeyCredentialsProvider(
+      const std::string& accessKey,
+      const std::string& secretKey) const {
+    return std::make_shared<Aws::Auth::SimpleAWSCredentialsProvider>(
+        awsString(accessKey), awsString(secretKey));
+  }
+
+  // Configure default AWSCredentialsProvider.
```

Comment:
What do you mean by "Configure"? There are not configuration parameters here.

Reply:
I guess `Build` is apt.